build.sbt

```diff
@@ -1,13 +1,20 @@
 import Dependencies._
 
 ThisBuild / organization := "org.constellation"
-ThisBuild / scalaVersion := "2.13.8"
-ThisBuild / version := "2.0.0"
+ThisBuild / scalaVersion := "2.13.10"
 ThisBuild / scalafixDependencies += Libraries.organizeImports
-ThisBuild / assemblyMergeStrategy in assembly := {
-  case PathList("META-INF", xs @ _*)            => MergeStrategy.discard
-  case PathList("org", "joda", "time", xs @ _*) => MergeStrategy.last
-  case x                                        => MergeStrategy.first
+ThisBuild / evictionErrorLevel := Level.Warn
+
+val ghTokenSource = TokenSource.GitConfig("github.token") || TokenSource.Environment("GITHUB_TOKEN")
```

Comment:
Maybe just assign straight to `githubTokenSource` without this intermediary val? It's not reused anywhere if I checked correctly.

Reply:
Makes sense.